rest_blog/tasks.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import, unicode_literals
+
+import os
+from os import name
+
+from celery import Celery
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.mail import send_mail
+# set the default Django settings module for the 'celery' program.
+from celery.task import periodic_task
+
+from blog.models import Reminder
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'Reminder.settings')
+
+app = Celery('Reminder', backend='amqp', broker='amqp://')
+app.config_from_object('Reminder.settings')
```

Comment:
I don't think you need to repeat all this again. You can just call `from celery.decorators import task`. And use it as decorator `@task(name=''send_mail')`.